app/lib/govuk_notify_personalisation.rb

```diff
@@ -200,6 +202,92 @@ def mmr_second_dose_message
     ].join("\n\n")
   end
 
+  def mmr_second_dose_required
+    mmr_programme.present? && patient_eligible_for_additional_dose?
+  end
+
+  def invitation_to_clinic_generic_message
+    [
+      (
+        if mmr_second_dose_required
+          "If you would like your local GP surgery to give #{short_patient_name} " \
+            "their 2nd dose, contact the surgery in the usual way."
+        end
+      ),
+      "#{mmr_second_dose_required ? "Alternatively, they" : "They"} can have this vaccination " \
+        "at a community clinic. If you’d like to book a clinic appointment, please contact " \
+        "us using the details below.",
+      (mmr_second_dose_waiting_period_message if mmr_second_dose_required)
+    ].compact.join("\n\n")
+  end
+
+  def mmr_invitation_to_clinic_message
+    return "" unless mmr_second_dose_required
+
+    case session&.organisation&.ods_code
```

Comment:
Do we know if any of the other teams are going to need these messages? I'm wondering if we could have an attribute on the team rather than needing to look up the ODS codes?

That way if we needed to apply it to more teams we could do so in the database without needing a code change.